tensorflow/core/util/tensor_bundle/tensor_bundle.cc

```diff
@@ -400,24 +401,31 @@ BundleWriter::BundleWriter(Env* env, StringPiece prefix, const Options& options)
     : env_(env),
       options_(options),
       prefix_(prefix),
-      tmp_metadata_path_(strings::StrCat(MetaFilename(prefix_), ".tempstate",
-                                         random::New64())),
-      tmp_data_path_(strings::StrCat(DataFilename(prefix_, 0, 1), ".tempstate",
-                                     random::New64())),
       out_(nullptr),
       size_(0) {
+  env_->HasAtomicMove(prefix_, &use_temp_file_);
```

Comment:
This results in an error under strict checking:

```
tensorflow/core/util/tensor_bundle/tensor_bundle.cc:406:3: error: ignoring return value of function declared with 'warn_unused_result' attribute [-Werror,-Wunused-result]
  env_->HasAtomicMove(prefix_, &use_temp_file_);
  ^~~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~
```

Please fix.

Reply:
Fixed it